igor-web/src/main/java/com/netflix/spinnaker/igor/scm/ManagedDeliveryScmService.java

```diff
@@ -63,10 +61,8 @@ public ManagedDeliveryScmService(
     this.gitHubMaster = gitHubMaster;
     this.gitLabMaster = gitLabMaster;
     this.bitBucketMaster = bitBucketMaster;
-    this.jsonMapper = jsonMapper;
-    this.yamlMapper =
-        new ObjectMapper(new YAMLFactory())
-            .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+    this.jsonMapper = new ObjectMapper();
+    this.yamlMapper = new ObjectMapper(new YAMLFactory());
```

Comment:
I decided to instantiate my own `ObjectMappers` here to avoid any unexpected influence from configuration elsewhere.